.goreleaser.yml

```diff
@@ -76,3 +87,23 @@ scoop:
   description: Command-line interface for SecretHub
 
   license: Apache-2.0
+
+nfpms:
+  -
+    builds:
+      - without-bin-dir
+    vendor: SecretHub
+    homepage: https://secrethub.io/
```

Comment:
`secrethub.io` or `secrethub.io/`? Let's make it consistent with the other urls in this file.

Reply:
It's `https://secrethub.io`. Changed it.